crates/lang/tests/ui/contract/fail/message-selector-invalid-type-01.stderr

```diff
@@ -1,4 +1,4 @@
-error: #[ink(selector = ..)] attributes with string inputs are deprecated. use an integer instead, e.g. #[ink(selector = 1)] or #[ink(selector = 0xC0DECAFE)].
+error: #[ink(selector = ..)] attributes with string inputs are deprecated (except for the wildcard selector "_"). use an integer instead, e.g. #[ink(selector = 1)] or #[ink(selector = 0xC0DECAFE)].
```

Comment:
Yeah, so the issue is [here](https://github.com/paritytech/ink/blob/master/crates/lang/ir/src/ir/attrs.rs#L809). We would need access to `_` there.

But the crate which we use for parsing attributes a la `key = "value"` does not support parsing verbatims such as `_`. Specifically the type `syn::Meta::NameValue` which we use as part of the [`syn::parse_meta`](https://docs.rs/syn/1.0.81/syn/struct.Attribute.html#method.parse_meta) return value does not support this.

I briefly looked into `syn::parse_meta`, but it was nothing trivial. Maybe we could also do some pre-processing, so that `selector = _` is transformed into `selector = "_"` before passing it to the lib, but I haven't looked into that.

Reply:
Or you can implement your own `parse_meta`=) You can copy the main logic from `syn` + add support of `_`=)